python/pyspark/streaming/tests.py

```diff
@@ -1549,7 +1549,9 @@ def search_kinesis_asl_assembly_jar():
         kinesis_jar_present = True
         jars = "%s,%s,%s" % (kafka_assembly_jar, flume_assembly_jar, kinesis_asl_assembly_jar)
 
-    os.environ["PYSPARK_SUBMIT_ARGS"] = "--jars %s pyspark-shell" % jars
+    existing_args = os.environ.get("PYSPARK_SUBMIT_ARGS")
```

Comment:
@vanzin, I think this will be broken if `PYSPARK_SUBMIT_ARGS` is not set and it returns `None`.

Reply:
I think there could be ... at least I know one informal way like `SPARK_TESTING=1 pyspark pyspark.sql.tests` for a quicker test. It wouldn't hard to have one if statement.

Reply:
or get("", ...) if possible